source/TimePicker.js

```diff
@@ -75,13 +122,13 @@
 		* @private
 		*/
 		setupItem: function (inSender, inEvent) {
-			var index = inEvent.index % this.range;
-			this.$.item.setContent(this.meridiems[index]);
+			var index = inEvent.index % this.range || 0;
```

Comment:
Do you remember why you had to make this change? Will `this.range` ever be a non-zero falsy value?